probables/exceptions.py

```diff
@@ -43,7 +43,7 @@ class CuckooFilterFullError(ProbablesBaseException):
     Args:
         message (str): The error message to be reported"""
 
-    def __init__(self, message: str):
+    def __init__(self, message: str) -> str:
```

Comment:
I believe this should be `-> None` and not `-> str`

Reply:
You are right,